catalog/core/catalog-core-directorymonitor/src/test/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitorTest.java

```diff
@@ -22,322 +22,196 @@
+    @Rule
+    public TemporaryFolder temporaryFolder = new TemporaryFolder();
 
-        verifyRoute(routeDefinition, MONITORED_DIRECTORY, copyIngestedFiles);
-    }
-
-    @Test
-    public void testRouteCreationMissingMonitoredDirectory() throws Exception {
-        String monitoredDirectory = "";
-        boolean copyIngestedFiles = true;
+    @Before
+    public void setup() throws Exception {
+        monitoredDirectory = temporaryFolder.newFolder("inbox");
+        monitoredDirectoryPath = monitoredDirectory.getAbsolutePath();
 
-        camelContext = (ModelCamelContext) super.createCamelContext();
+        camelContext = super.createCamelContext();
         camelContext.start();
 
-        // Map the "content" scheme to a mock component so that we do not have to
-        // mock the entire custom ContentComponent and include its implementation
-        // in pom with scope=test
-        camelContext.addComponent("content", new MockComponent());
-
-        contentDirectoryMonitor = new ContentDirectoryMonitor(camelContext);
-        contentDirectoryMonitor.systemSubjectBinder = noOpProcessor;
-        contentDirectoryMonitor.setMonitoredDirectoryPath(monitoredDirectory);
-        contentDirectoryMonitor.setCopyIngestedFiles(copyIngestedFiles);
-
-        // Simulates what container would do once all setters have been invoked
-        contentDirectoryMonitor.init();
+        MockComponent contentComponent = new MockComponent();
+        camelContext.addComponent("content", contentComponent);
+    }
 
-        assertThat(camelContext.getRouteDefinitions()
-                .size(), is(0));
+    @After
+    public void destroy() throws Exception {
+        camelContext.stop();
     }
 
     @Test
-    public void testMoveFolder() throws Exception {
-        boolean copyIngestedFiles = true;
-
-        createRoute(MONITORED_DIRECTORY, copyIngestedFiles);
-
-        // Put file in monitored directory
-        String fileContents = "Dummy data in a text file";
-        FileUtils.writeStringToFile(new File(INPUT_FILEPATH), fileContents);
-
-        template.sendBodyAndHeader("file://" + MONITORED_DIRECTORY,
-                fileContents,
-                Exchange.FILE_NAME,
-                INPUT_FILENAME);
-
-        Thread.sleep(3000);
-
-        // Verify the file moved to the .ingested directory
-        File target = new File(MONITORED_DIRECTORY + "/.ingested/" + INPUT_FILENAME);
-        assertTrue("File not moved to .ingested folder", target.exists());
-
-        // Cleanup
-        FileUtils.deleteDirectory(new File(MONITORED_DIRECTORY));
+    public void testRouteCreationWithoutContentComponent() throws Exception {
+        camelContext.removeComponent("content");
+        ContentDirectoryMonitor monitor = createContentDirectoryMonitor();
+        submitConfigOptions(monitor, monitoredDirectoryPath, false);
+        assertTrue(monitor.getRouteDefinitions()
```

Comment:
I think assertThat is preferred here : `assertThat(monitor.getRouteDefinitions(), hasSize(0))`